### ISSUE TITLE
Chiller:Electric:EIR AirCooled condenser outlet node conditions not correct

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5629]
## Issue overview

User file reports condenser outlet node temperatures on the order of 4000C for a Chiller:Electric:EIR with AirCooled condenser.  I expected the chiller to ignore the condenser outlet node name (with a warning) for AirCooled condenser, but it apparently is calculating conditions for this node.  Also, noticed this in the eio output, even though condenser water volume flow rate is autosized:

`Component Sizing Information, Chiller:Electric:EIR, REFRIGERATORE, User-Specified Reference Condenser Water Flow Rate [m3/s], 1.10000E-003`

So, looks like a flow rate that's more appropriate for water is being used but with the density and specific heat of air, resulting in very high condenser outlet node temperatures.  This doesn't really impact any oter results, but it is confusing.  This chiller (and others?) should either warn and ignore the condenser outlet node name, or be fixed to autosize a condenser air flow rate and calculate a proper outlet condition.
### Details

Some additional details for this issue (if relevant):
- Platform Win64
- Version of EnergyPlus 8.5.0
- Helpdesk ticket 11301
### Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [ ] Defect file added (list location of defect file here)
- [ ] Ticket added to Pivotal for defect (development team task)
- [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by rraustad** [comment]
Verified defect as shown:

![5629-1](https://cloud.githubusercontent.com/assets/5778156/16957180/65de0514-4da9-11e6-9cc0-8deb8db716d3.png)

And corrected version:

![5629-2](https://cloud.githubusercontent.com/assets/5778156/16957194/6f2b124c-4da9-11e6-96c2-11f30d83bb13.png)

**Comment by rraustad** [comment]
Included report variables for water use rate. Haven't connected to a storage tank since that would require a new input (Supply Water Storage Tank Name). Also need to add new inputs for Evaporative Condenser Effectiveness and Evaporative Condenser Pump Rated Power Consumption. This seems beyond the scope of this issue.

![5629_3](https://cloud.githubusercontent.com/assets/5778156/16960132/2d6ef0dc-4db6-11e6-8849-99a0d390f421.png)

**Comment by mjwitte** [comment]
@rraustad You say "Included report variables for water use rate".  Does that mean current code does not consume any water if the condenser is evap cooled?

**Comment by rraustad** [comment]
See PR #5782

**Comment by rraustad** [comment]
I believe so since the report variables were not set up to count the water use. I added:

```
SetupOutputVariable( "Chiller Evaporative Condenser Water Volume [m3]", ElectricEIRChillerReport( EIRChillerNum ).EvapWaterConsump, "System", "Sum", ElectricEIRChiller( EIRChillerNum ).Name, _, "Water", "Cooling", _, "System" );
SetupOutputVariable( "Chiller Evaporative Condenser Mains Supply Water Volume [m3]", ElectricEIRChillerReport( EIRChillerNum ).EvapWaterConsump, "System", "Sum", ElectricEIRChiller( EIRChillerNum ).Name, _, "MainsWater", "Cooling", _, "System" );
```

**Comment by mjwitte** [comment]
OK.  Then please make sure the addition of water consumption is highlighted in the change description.

**Comment by rraustad** [comment]
Water use summary for V8.5 and develop in tab report.

V8.5 (values are [m3] [%]):

Water Source Summary 
    Water Supplied by Utility   0.00    -

Develop:

Water Source Summary
    Water Supplied by Utility   4594.70 100.00

Similar for html report (V8.5 showed 0), building area = 19.5 m2 x 235.66 m3/m2 = 4595.37 m3

![5629-html](https://cloud.githubusercontent.com/assets/5778156/16988514/a4c23722-4e5e-11e6-95ea-8bdc35e6dd0b.png)

**Comment by Myoldmopar** [comment]
Closed via #5782
